package-lock.json

```diff
@@ -1,1253 +1,9219 @@
 {
   "name": "wikit",
+  "lockfileVersion": 2,
```

Comment:
Sorry for being nit-picky here, but it seems you ran `npm install` here not with docker, but with some local npm? Please regenerate the package-lock.json for this commit with `docker-compose run --rm --user=$(id -u):$(id -g) node npm install --package-lock-only`

Reply:
The reasoning behind my request is that this would otherwise cause local changes to the `package-lock.json` whenever runs `npm install` with the docker command locally. That isn't a great developer experience.

Reply:
I think you got a point... I did run `npm install` locally. Lemme fix that